packages/bodiless-demo/src/shadow/@bodiless/vital-card/FlowContainer.ts

```diff
@@ -0,0 +1,13 @@
+import { as } from '@bodiless/fclasses';
+import { vitalCardFlowContainerBase } from '@bodiless/vital-card';
+
+// Demostrating only showing Basic & Hero Card Variations in Component Picker
```

Comment:
What cards are we excluding?  Is this in order to ensure the "orientation" facet always appears?  Are there cards in Vital which don't hve this facet?  If so, can we add it there?

Reply:
Can do so.